Fibonacci.java

```diff
@@ -1,25 +0,0 @@
-/*
-Fibonacci serie - defined by array 10 elements; program can show a
-selected number of serie for example position three: 1
-*/
-
-import java.util.Scanner;
-
-public class Fibonacci {
-	public static void main(String[] args) {
-	
-	int fibonacciSeries[] = {0,1,1,2,3,5,8,13,21,34};
-	
-	System.out.println("write a number in serie (1...10): ");
-	Scanner input = new Scanner(System.in);
-	
-	int intUserInput = input.nextInt();
-	int serieArrayNumber = intUserInput - 1;
-	
-	System.out.print("value of " + intUserInput + " fibonacci serie: ");
-	System.out.println(fibonacciSeries[serieArrayNumber]);
-	
-	System.out.print("value of " + intUserInput + " fibonacci series: ");
-	System.out.println(fibonacciSeries[intUserInput]);
-	}
-}
```

Comment:
Hey @lukaszkutylowski 
Do you understand what's going on here?
Please, explain if this was intentional.
If not, then please correct.
If yes - than please explain it to me.
Use @ sign to mention me somewhere or just call If that's easier

Reply:
Hey Man!
Why you didn't answer?
:)

Don't be shy, reply:)